curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/61764c602bee6974e7790f35.md

```diff
@@ -13,6 +13,14 @@ These days, there are two main color models: the <dfn>additive</dfn> RGB (red, g
 
 First, add the `class` `one` to the first marker `div` element.
 
+Here is the syntax to add multiple classes:
+
+```html
+<div class="class1 class2">
```

Comment:
I think it's better to not use `class1 class2`, because people may change `class=""`

```diff
- <div class="class1 class2">
+ <div class="exampleClass1 exampleClass2">
```